src/mqt/bench/devices/ionq.py

```diff
@@ -6,91 +6,75 @@
-            self.ionq_calibration = cast("IonQCalibration", json.load(json_file))
-        self.calibration = None
-
-        self.name = self.ionq_calibration["name"]
-        self.gateset = Gateset("ionq", self.ionq_calibration["basis_gates"])
-        self.num_qubits = self.ionq_calibration["num_qubits"]
-        self.coupling_map = list(self.ionq_calibration["connectivity"])
-
-    def read_calibration(self) -> None:
-        """Read the calibration data for the device."""
-        calibration = DeviceCalibration()
-        for qubit in range(self.num_qubits):
-            calibration.single_qubit_gate_fidelity[qubit] = dict.fromkeys(
-                ["ry", "rx"], self.ionq_calibration["fidelity"]["1q"]["mean"]
-            )
-            calibration.single_qubit_gate_fidelity[qubit]["rz"] = 1  # rz is always perfect
-            calibration.single_qubit_gate_duration[qubit] = dict.fromkeys(
-                ["ry", "rx"], self.ionq_calibration["timing"]["1q"]
-            )
-            calibration.single_qubit_gate_duration[qubit]["rz"] = 0  # rz is always instantaneous
-            calibration.readout_fidelity[qubit] = self.ionq_calibration["fidelity"]["spam"]["mean"]
-            calibration.readout_duration[qubit] = self.ionq_calibration["timing"]["readout"]
-            calibration.t1[qubit] = self.ionq_calibration["timing"]["t1"]
-            calibration.t2[qubit] = self.ionq_calibration["timing"]["t2"]
-
-        for qubit1, qubit2 in self.coupling_map:
-            calibration.two_qubit_gate_fidelity[qubit1, qubit2] = {
-                "rxx": self.ionq_calibration["fidelity"]["2q"]["mean"]
-            }
-            calibration.two_qubit_gate_duration[qubit1, qubit2] = {"rxx": self.ionq_calibration["timing"]["2q"]}
-        self.calibration = calibration
-
-
-class IonQHarmony(IonQDevice):
-    """IonQ Harmony device."""
-
-    def __init__(self) -> None:
-        """Initialize the IonQ Harmony device."""
-        super().__init__(get_device_calibration_path("ionq_harmony"))
-
+def create_ionq_target(calibration_path: Path) -> Target:
+    """Create a target device from the IonQ calibration data."""
+    with calibration_path.open() as json_file:
+        calib = json.load(json_file)
 
-class IonQAria1(IonQDevice):
-    """IonQ Aria1 device."""
+    target = Target(num_qubits=calib["num_qubits"], description=calib["name"])
+    num_qubits = calib["num_qubits"]
+    connectivity = calib["connectivity"]
 
-    def __init__(self) -> None:
-        """Initialize the IonQ Aria1 device."""
-        super().__init__(get_device_calibration_path("ionq_aria1"))
+    # Gate durations and fidelities
+    oneq_fidelity = calib["fidelity"]["1q"]["mean"]
+    twoq_fidelity = calib["fidelity"]["2q"]["mean"]
+    spam_fidelity = calib["fidelity"]["spam"]["mean"]
 
+    oneq_duration = calib["timing"]["1q"]
+    twoq_duration = calib["timing"]["2q"]
+    readout_duration = calib["timing"]["readout"]
```

Comment:
I would be very tempted to simply hard code the calibration file values here.
We have never updated them. Since their creation. And they are fairly specific to a particular device.
We could simply link to the origin of the data.
Or, wild and crazy idea: Simply remove all the calibration information for now to streamline the implementation and then come up with a good strategy to selectively add it back wherever feasible.
MQT Bench was not using that information so far anyway.

Reply:
I like the idea and would like to discuss the consequences in terms of MQT Predictor in person. Let's then decide based on that discussion.